### DISCUSSION TITLE
Enable folders to better organise lots of pdfs

### DISCUSSION THREAD

**Discussion by Splinter7149** [sumatrapdfreader/sumatrapdf#3302] [Ideas]
People who have lots of PDFs open will agree with me, organising a bunch of PDFs are a pain. If only there were folders which we could sort them into and open each folder at a time.

**Comment by kjk** [comment]
I've been thinking about implementing something like metapath in notepad2 / notepad3:

![image](https://user-images.githubusercontent.com/7935/221343051-00d9a915-9b91-415a-b3a8-7f358843dc9a.png)

The idea is to create a specialized file browser which only shows files supported by Sumatra and opening them directly in SumatraPDF.

There would be an easy to bookmark folders / files for quick access.
